### DISCUSSION TITLE
No module named 'line_chatbot_api

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#310] [Q&A]
大家好，在寫HW2時我程式碼改好了，想嘗試將它放進我的line bot 跑看看結果，但目前顯示以下錯誤，不知道有沒有大大能夠協助我解決
![225335](https://user-images.githubusercontent.com/90717869/168544894-cf464ec5-91b9-409e-a425-502d4c4e7c94.jpg)
?

**Comment by chia-kai-chang** [comment]
應該要執行 `app.py` ， 而不是 `line_chatbot_api.py`
